### DISCUSSION TITLE
想问一下如果我想改变默认的那些蓝色（按钮啊，提示之类的），应该怎么操作？

### DISCUSSION THREAD

**Discussion by uedkkk** [liuyib/hexo-theme-stun#176] [Q&A（反馈问题、提问、报告 Bug 等）] [answered]
rt

**Comment by liuyib** [comment, marked as answer]
> 首先抱歉这么晚回复，最近在忙于毕设。

目前还没有统一的配置可以实现，不过你可以通过修改源码中的[全局 CSS 变量](https://github.com/liuyib/hexo-theme-stun/blob/master/source/css/_variables/index.styl)来暂时实现。感谢反馈 ❤️ ，以后会提供这个需求的相关配置和文档。
